File: 프로그래머스/1/12903. 가운데 글자 가져오기/가운데 글자 가져오기.js
function solution(s) {
    
    // 문자열의 길이
    const len = s.length;
    
    // 짝수/홀수 구분 
    // 짝수인 경우 
    if(len % 2 === 0){
        const mid = len / 2;
         return s[mid - 1] + s[mid];
    } else {
        // 홀수인 경우
        return s[Math.floor(len / 2)];
    }
    
}

File: 프로그래머스/1/12903. 가운데 글자 가져오기/가운데 글자 가져오기.py
def solution(s):
    print(len(s)//2)
    
    if len(s) % 2 == 0:
        return s[(len(s) // 2) - 1:(len(s) // 2) +1]
    else: 
        return s[(len(s) // 2)]